sickbeard/manual_search.py

```diff
@@ -92,14 +92,15 @@ def getEpisodes(search_thread, searchstatus):
             search_thread.segment = [search_thread.segment]
 
     for ep_obj in search_thread.segment:
-        results.append({'show': ep_obj.show.indexerid,
-                        'episode': ep_obj.episode,
-                        'episodeindexid': ep_obj.indexerid,
-                        'season': ep_obj.season,
+        ep = show_obj.getEpisode(ep_obj.season, ep_obj.episode)
+        results.append({'show': show_obj.indexerid,
+                        'episode': ep.episode,
+                        'episodeindexid': ep.indexerid,
+                        'season': ep.season,
                         'searchstatus': searchstatus,
-                        'status': statusStrings[ep_obj.status],
-                        'quality': getQualityClass(ep_obj),
-                        'overview': Overview.overviewStrings[show_obj.getOverview(ep_obj.status)]})
+                        'status': statusStrings[ep.status],
+                        'quality': getQualityClass(ep),
+                        'overview': Overview.overviewStrings[show_obj.getOverview(ep.status)]})
```

Comment:
Style change here.  I would do:
```python
results.append({
    'show': show_obj.indexerid,
    'episode': ep.episode,
    'episodeindexid': ep.indexerid,
    'season': ep.season,
    'searchstatus': searchstatus,
    'status': statusStrings[ep.status],
    'quality': getQualityClass(ep),
    'overview': Overview.overviewStrings[show_obj.getOverview(ep.status)],
})
```
That way if the variable `results` changes, indentation of dict keys stays constant.  Also note the `,` after the final key.  It makes it easy to add additional keys later.